src/geometry/size.rs

```diff
@@ -19,7 +19,7 @@ impl Size {
     }
 
     /// Returns true if the `Point` is contained in this `Size` or false otherwise
-    pub fn contains(&self, point: Point) -> bool {
+    pub fn _contains(&self, point: Point) -> bool {
```

Comment:
Why the underscore? Is the function no longer in use?

Reply:
Yes, it is no longer in use. But it looks useful. Should I just remove it?

Reply:
Yes, we can always rescue it from the git history if necessary